Track.h

```diff
@@ -109,12 +127,20 @@ class Track
 private:
   TrackState state_;
   HitVec hits_;
-  HitVec initHits_;
-  HitIdxVec hitIdxVec_;
   float chi2_;
+  unsigned int mcTrackID_;
+  unsigned int nHitsMatched_;
+  unsigned int seedID_;
+  unsigned int duplicateID_;
+  bool isDuplicate_;
+  HitIdxVec hitIdxVec_;
```

Comment:
you are actually adding private members to track, this is not good as it will increase the size. 
we should merge asap also the data format reduction from my branches